mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/AttachmentLoader.java

```diff
@@ -40,22 +41,22 @@ public AttachmentLoader(AttachmentMapper attachmentMapper) {
         this.attachmentMapper = attachmentMapper;
     }
 
-    public Collection<org.apache.james.mailbox.model.MessageAttachment> getAttachments(Set<CassandraMessageDAO.MessageAttachment> attachmentsById) {
+    public Collection<MessageAttachment> getAttachments(Set<CassandraMessageDAO.MessageAttachmentRepresentation> attachmentsById) {
```

Comment:
Sorry my renaming was more about the Set<CassandraMessageDAO.MessageAttachmentRepresentation> attachmentsById stuff. Globally attachmentsById is too more used in this class, without the expected meaning.